src/menus/EditMenus.cpp

```diff
@@ -738,8 +738,20 @@ void OnSilence(const CommandContext &context)
    auto &tracks = TrackList::Get(project);
    auto &selectedRegion = ViewInfo::Get(project).selectedRegion;
 
-   for (auto n : tracks.Selected<WaveTrack>())
-      n->Silence(selectedRegion.t0(), selectedRegion.t1());
+   const auto selectedWaveTracks = tracks.Selected<WaveTrack>();
+   auto progress = BasicUI::MakeProgress(
+      XO("Pre-processing"), XO("Rendering Time-Stretched Audio"));
```

Comment:
New strings.  @LWinterberg

Reply:
FYI copy-paste of agreed-upon strings from other places where pre-rendering is needed.